UpdatedImageProcessing/Classifiers/classify_color.py

```diff
@@ -0,0 +1,256 @@
+import sys
+from os import walk
+import cv2
+import numpy as np
```

Comment:
We are trying to get away from this syntax. Please just use numpy, then rename all uses of np to numpy.

Reply:
Do we want to move away from the `import numpy as np` convention? In other scripts that are math heavy, I imagine that renaming all `np` to `numpy` would make it harder to read.